Podfile

```diff
@@ -6,6 +6,10 @@ source 'https://github.com/CocoaPods/Specs.git'
 
 target :Weather do
   pod 'Swinject'
+  pod 'Alamofire'
+  pod 'ModelMapper'
+  pod 'ASToast'
+  pod 'RealmSwift'
 end
 
 target :WeatherTests do
```

Comment:
we need this in the test target as it won't link it otherwise

  pod 'RealmSwift'